Tests/KlaviyoCoreTests/__Snapshots__/EncodableTests/testEventPayloadWithoutMetadata.1.json

```diff
@@ -4,7 +4,7 @@
       "metric" : {
         "data" : {
           "attributes" : {
-            "name" : "blob"
+            "name" : "test"
```

Comment:
Maybe a little weird...did you change this test data?

Reply:
Yeah looks like,  looks like when I switched out the models I used a different name - 

Older tests 

```swift
func testEventPayloadWithoutMetadata() throws {
    let event = Event.test
    let createEventPayload = KlaviyoAPI.KlaviyoRequest.KlaviyoEndpoint.CreateEventPayload(data: .init(event: event, anonymousId: "anon-id"))
    assertSnapshot(matching: createEventPayload, as: .json(KlaviyoEnvironment.encoder))
}


extension Event {
    static let SAMPLE_PROPERTIES = [
        "blob": "blob",
        "stuff": 2,
        "hello": [
            "sub": "dict"
        ],
        "Application ID": "com.klaviyo.fooapp",
        "App Version": "1.2.3",
        "App Build": "1",
        "App Name": "FooApp",
        "OS Version": "1.1.1",
        "OS Name": "iOS",
        "Device Manufacturer": "Orange",
        "Device Model": "jPhone 1,1"
    ] as [String: Any]
    static let test = Self(name: .CustomEvent("blob"), properties: SAMPLE_PROPERTIES)
}
```

Current tests 

```swift
    func testEventPayloadWithoutMetadata() throws {
        let createEventPayload = CreateEventPayload(data: CreateEventPayload.Event(name: "test", anonymousId: "anon-id"))
        assertSnapshot(matching: createEventPayload, as: .json(KlaviyoEnvironment.encoder))
    }
```